### ISSUE TITLE
NameError when trying to run Websockets-example

### ISSUE THREAD

**Issue by ankile** [vibora-io/vibora#134]
Hi,
I'm trying to run the simple websocket-example and I'm getting a NameError:
NameError: name 'Request' is not defined

My code is as follows:

from vibora import Vibora
from vibora.websockets import WebsocketHandler

app = Vibora()

@app.websocket(b"/")
class ConnectedClient(WebsocketHandler):
    async def on_message(self, msg):
        print(msg)
        await self.send(msg)

    async def on_connect(self):
        print("Client connected")

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=8000, debug=True)

I have cloned the repo, installed requirements.txt and run build.py.
I really hope there's a easy fix for this because this framework seems perfect for me if I can only get some simple websockets up and running :)

**Comment by danieldaeschle** [comment]
Please post the full stack trace.

**Comment by ankile** [comment]
Yes, of course, sorry about that, it's in the original post now.

**Comment by danieldaeschle** [comment]
I just tried to run the example and got the same error. I don't find the bug. We have to wait what @frnkvieira says :)

**Comment by shikhir-arora** [comment]
I haven't tried it but this is probably due to there being [no `__all__`](https://github.com/vibora-io/vibora/blob/master/vibora/websockets/__init__.py) in the `__init__` file for `vibora.websockets`, though it [looks like it was fixed in this branch](https://github.com/vibora-io/vibora/blob/v0.1.0/vibora/websockets/__init__.py). I haven't tried it though, so there may be other issues.
